src/backend/tests/conftest.py

```diff
@@ -75,6 +75,12 @@ def blockbuster(request):
                 .can_block_in("langchain_core/_api/internal.py", "is_caller_internal")
             )
 
+            for func in ["os.stat", "os.path.abspath", "os.scandir"]:
+                bb.functions[func].can_block_in("alembic/util/pyfiles.py", "load_python_file")
```

Comment:
Alembic makes a few blocking os calls. These shouldn't have a big impact as thses blockings are short and migrations happen only at startup.